draft-ietf-add-ddr.md

```diff
@@ -91,8 +91,8 @@ Both of these approaches allow clients to confirm that a discovered Encrypted
 Resolver is designated by the originally provisioned resolver. "Designated" in
 this context means that the resolvers are operated by the same entity or
 cooperating entities; for example, the resolvers are accessible on the same
-IP address, or there is a certificate that claims ownership over the designated
-resolver IP address (regardless of what IP the designated resolver is on).
+IP address, or there is a certificate that claims ownership over the original
```

Comment:
Good catch Tommy.

We've lost the distinction of the ownership being over the IP address (as opposed to any other property of the designating resolver, such as a name). I suggest inserting "IP address of the" before "original"